static/style.css

```diff
@@ -1554,14 +1558,20 @@ code .comment {
   height: 320px;
   margin: 16px 0;
 
-  border-radius: 4px 2px 8px;
+  border-radius: 4px 2px 4px;
   overflow: hidden;
 
   user-select: none;
+
+  --scrollbar-width: 14px;
 }
 
 .flame-chart-hovered-block-info {
-  height: 64px;
+  height: 96px;
+}
+
+.flame-chart-hovered-block-info .data {
+  color: var(--body-text-color);
```

Comment:
feels simpler to just do #000 here since it's only used in one place

Reply:
Fair enough, if we do want CSS vars for the body text color then that can be done in its own PR

Reply:
yeah, i feel like if we want to use variables we should device if we want to use sass or something else